multiversxetl/jobs/load_job.py

```diff
@@ -3,6 +3,9 @@
 
 from google.cloud import bigquery
 
+WRITE_DISPOSITION_APPEND = "WRITE_APPEND"
```

Comment:
please also use them in the test file :D

Reply:
Fixed.